cmd/timoni/bundle_build_test.go

```diff
@@ -31,35 +30,66 @@ func Test_BundleBuild(t *testing.T) {
 	))
 	g.Expect(err).ToNot(HaveOccurred())
 
-	bundleData := fmt.Sprintf(`
+	bundleCue := fmt.Sprintf(`
 appName: string @timoni(env:string:TEST_BBUILD_NAME)
 bundle: {
 	apiVersion: "v1alpha1"
-	name: "%[1]s"
+	name: string
 	instances: {
 		"\(appName)": {
 			module: {
-				url:     "oci://%[2]s"
-				version: "%[3]s"
+				url:     "oci://%[1]s"
+				version: "%[2]s"
 			}
-			namespace: "%[4]s"
+			namespace: "%[3]s"
 			values: server: enabled: false
 			values: domain: string @timoni(env:string:TEST_BBUILD_HOST)
 		}
 		backend: {
 			module: {
-				url:     "oci://%[2]s"
-				version: "%[3]s"
+				url:     "oci://%[1]s"
+				version: "%[2]s"
 			}
-			namespace: "%[4]s"
+			namespace: string
 			values: client: enabled: bool @timoni(env:bool:TEST_BBUILD_ENABLED)
 		}
 	}
 }
-`, bundleName, modURL, modVer, namespace)
+`, modURL, modVer, namespace)
 
-	bundlePath := filepath.Join(t.TempDir(), "bundle.cue")
-	err = os.WriteFile(bundlePath, []byte(bundleData), 0644)
+	bundleData := bundleCue + fmt.Sprintf(`
+bundle: name: "%[1]s"
+bundle: instances: backend: namespace: "%[2]s"
+`, bundleName, namespace)
+
+	bundleJson := fmt.Sprintf(`
+{
+	"bundle": {
+		"name": "%[1]s"
+	}
+}
+`, bundleName)
+
+	bundleYaml := fmt.Sprintf(`
+bundle:
+  instances:
+    backend:
+      namespace: %[1]s
+`, namespace)
+
+	cuef, err := ioutil.TempFile(t.TempDir(), "*.cue")
```

Comment:
`ioutil` is deprecated since Go 1.17, this should be `os.CreateTemp` from top of head.

Reply:
Good catch, should be like we do it elsewhere:

```
	bundlePath := filepath.Join(t.TempDir(), "bundle.cue")
	err = os.WriteFile(bundlePath, []byte(bundleData), 0644)
	g.Expect(err).ToNot(HaveOccurred())
```

`t.TempDir()` does a cleanup so we don't leave stuff in tmp.

Reply:
Oh nice catch! Will address it, thanks

Reply:
Done